src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
```

Comment:
Since this component is designed with a dark background, we'll want all the text to be light-colored. Since `color` is an inherited property, you should set this on the top-level element of this component so that all child elements inherit it.